src/model/mod.rs

```diff
@@ -1,6 +1,7 @@
 // Copyright 2022 CeresDB Project Authors. Licensed under Apache-2.0.
 
 pub mod convert;
+pub(crate) mod request;
```

Comment:
The request mod should be used by caller so should it be `pub mod request`?

Reply:
done